svelte/src/components/tabs/tabs.svelte

```diff
@@ -1,50 +1,35 @@
 <script lang="ts">
   import type { Tab } from "$libs/types";
-  import { afterUpdate } from "svelte";
+  import { tabStore } from "$libs/stores";
+  import Button from "../button/button.svelte";
 
-  let clazz = "";
+  const { activeTab, setActiveTab } = tabStore;
 
+  let clazz = "";
   export { clazz as class };
-
-  import Button from "../button/button.svelte";
-
   export let tabs: Tab[] = [];
-  export let defaultTab: string;
-
-  let active: string;
-
-  let dirty = false;
 
-  afterUpdate(() => {
-    if (tabs.length && !active) {
-      if (!defaultTab) {
-        active = tabs[0].label;
-      } else if (!dirty) {
-        active = defaultTab;
-      }
-    }
-  });
+  $: activeTabId = $activeTab ?? tabs[0]?.id;
```

Comment:
@ABevier still breaks after visiting other packages. ie recreation step:

1. go to SD, start -> it opens the terminal tab
2. go to a different package w/o terminal

Reply:
check if $activeTab exists else tabs[0]

Reply:
though what if terminal is running for SD, and for the other package its not yet opened should it show terminal?

Reply:
okay, it's fixed now